client/web/src/enterprise/batches/list/BatchChangeNode.tsx

```diff
@@ -23,6 +23,30 @@ export interface BatchChangeNodeProps {
     displayNamespace: boolean
 }
 
+const getBadge = (state: BatchChangeState): JSX.Element => {
```

Comment:
This smells like a react component disguising as a function to me 😬  feel free to ignore though. 
```suggestion
const BatchChangeBadge: React.FunctionComponent<{state: BatchChangeState}> = ({ state }) => {
```